docs/universal-gateway/overview.mdx

```diff
@@ -4,58 +4,36 @@ sidebar_label: Overview
+Here are some of the most common use cases for the Universal Gateway, from the [examples collection](/universal-gateway/examples/):
 
-Learn more about Endpoints:
+- [**The front door pattern**](/universal-gateway/examples/front-door-pattern/): Set up a single public endpoint to serve as the centralized entrance to all upstream services.
+- [**Route to endpoints by geography**](/universal-gateway/examples/route-by-geography/): Forward requests based on IP geolocation data for improved latency or country-specific features.
+- [**Create identity-based rate limits**](/universal-gateway/examples/pre-tier-requests/): Pre-tier requests based on your packaging or pricing model.
+- [**Secure a public Minecraft server**](/universal-gateway/examples/minecraft/): Restrict server access to a specific set of IP addresses.
+- [**Intercept and rewrite headers**](/universal-gateway/examples/rewrite-headers-redirects/): Intercept 302 redirect headers to preserve UX and agent behavior.
 
-- [Bindings →](/universal-gateway/bindings/)
-- [Internal Endpoints →](/universal-gateway/internal-endpoints/)
-- [Public Endpoints →](/universal-gateway/public-endpoints/)
-- [Kubernetes Endpoints →](/universal-gateway/kubernetes-endpoints/)
+## What's next?
 
-## Load Balancing
-
-Endpoints Pools make load balancing dead simple.
-When your create two endpoints with the same URL (and binding), those endpoints automatically "pool" together and traffic to their URL is balanced among them.
-
-- [Learn more about Load Balancing with Endpoint Pools →](/universal-gateway/endpoint-pooling/)
-
-## TLS
-
-ngrok automatically handles TLS (SSL) termination and certificate management for you.
-There is typically nothing to setup, configure, or manage.
-
-- [Learn more about TLS Certificates →](/universal-gateway/tls-certificates/)
-- [Learn more about the TLS Termination →](/universal-gateway/tls-termination/)
-
-## Global Load Balancer
-
-ngrok's Global Load Balancer automatically improves the performance and resiliency of your applications by distributing traffic to the nearest healthy point of presence, measured by latency, from the perspective of the connecting client.
-
-- [Learn more about the Global Load Balancer →](/universal-gateway/global-load-balancer/)
-- [See ngrok's Global Points of Presence →](/universal-gateway/points-of-presence)
-
-## DDoS Protection
-
-ngrok automatically protects your applications with out-of-the-box protection from distributed denial-of-service (DDoS) attacks.
-
-- [Learn more about DDoS Protection →](/universal-gateway/ddos-protection)
+- Explore key concepts for working with the Universal Gateway such as [endpoint types](/universal-gateway/types/), [protocols](/universal-gateway/protocols/), [bindings](/universal-gateway/bindings/), and [pooling](/universal-gateway/endpoint-pooling/).
+- Check out the [Universal Gateway examples collection](/docs/universal-gateway/examples/) to see how to implement even more common use cases.
```

Comment:
feels like this should be first in the list, since it's showing some concrete examples